includes/site-settings.php

```diff
@@ -649,3 +649,59 @@ function klyp_generate_custom_values_for_nonadmin_post_types($field)
         ';
     });
 }
+
+/**
+ * For backwards compatability
+ * 
+ * Save the new API settings values against the old one so old code doesn't break
+ * 
+ * @param string $post_id The post id to save to
+ * @param string $menu_slug the menu slug the save action is coming from
+ */
+function threeequals_acf_save_options_page($post_id, $menu_slug)
+{
+    if ($menu_slug !== 'site-settings') {
+        return;     
+    }
+ 
+    // Check google maps is fully updated
+    $api_google_map = get_field('settings_api', 'options')['settings_api_google_map'];
```

Comment:
we don't need to do this as acf/options_page/save is triggered after the save, so this value will exist.

we will just need to update the old field the new one.

if new one is purposely left blank, so we can assume we want to remove the old one, so need to be updated to blank as well